.pre-commit-config.yaml

```diff
@@ -48,3 +48,12 @@ repos:
     hooks:
     -   id: mypy
         exclude: tests/
+-   repo: https://github.com/PrincetonUniversity/blocklint
+    rev: v0.2.4
+    hooks:
+    -   id: blocklint
+        args:
+            - 'docs'
+            - 'tests'
+            - '<MODULE_NAME>'
+            - '--skip-files docs/releases.rst'
```

Comment:
Still WIP here, not sure how args need to be passed in.

Reply:
You can use the `pre-commit exclude` option, as here: https://github.com/ethereum/eth-typing/pull/57/files#diff-63a9c44a44acf85fea213a857769990937107cf072831e1a26808cfde9d096b9

Reply:
if you want to explicitly include files, use `files`